docs/static/ea-integration-tutorial.asciidoc

```diff
@@ -0,0 +1,185 @@
+[[ea-integrations-tutorial]]
+== Using {ls} with Elastic {integrations} (Beta) tutorial
+
+
+Logstash elastic-integration Filter
+Plugin Guide
+Ingest node pipelines in Logstash
```

Comment:
Ingest node pipelines are an _internal_ implementation detail of the Elastic Integration filter. They are _how_ it currently works, and not _what_ they do or _why_ they do it.

I specifically want to avoid elevating this internal detail into an external commitment because of how _drastically_ it changes the expectations our users have. If users are starting out with the mindset that they can run arbitrary ingest node pipelines, and put in development effort to work around the various internal-implementation-detail limitations, and we later change our internal implementation to improve the _general_ experience of running Elastic Integrations, the users who pinned themselves to internal details will end up with a broken experience.

```suggestion
Elastic Integrations in Logstash
```